src/message/topic-descriptor.proto.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+module.exports = `
+// topicCID = cid(merkledag_protobuf(topicDescriptor)); (not the topic.name)
```

Comment:
I copied this file from `js-libp2p-floodsub`. I'm not sure what purpose of that line is. Would it be safe to remove it all together since it has nothing to do with the topic descriptor protocol buffer?

Reply:
let's remove it then in both projects!